main/webapp/modules/core/scripts/reconciliation/recon-dialog-2.js

```diff
@@ -0,0 +1,141 @@
+/*
+
+Copyright 2010, Google Inc.
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+ * Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above
+copyright notice, this list of conditions and the following disclaimer
+in the documentation and/or other materials provided with the
+distribution.
+ * Neither the name of Google Inc. nor the names of its
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,           
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY           
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+ */
+
+function ReconDialog2(column,selectedServiceRecordindex,serviceRecords,record) {
+  this._column = column;
+  this._serviceRecords = serviceRecords;
+  this._selectedServiceRecordIndex = selectedServiceRecordindex;
+  this._record=record;
+
+  this._createDialog();
+}
+
+ReconDialog2.prototype._createDialog = function() {
+  var self = this;
+  var dialog = $(DOM.loadHTML("core", "scripts/reconciliation/recon-dialog-2.html"));
+
+  this._elmts = DOM.bind(dialog);
+  this._elmts.dialogHeader.text($.i18n('core-recon/recon-col')+' "' + this._column.name + '"');
```

Comment:
This is something you just copied from `recon-dialog.js` and it's fine to do that, but it could be an occasion to fix the way we generate this text. We should not use concatenation here, but rather pass `this._column.name` as a parameter to the `$.i18n` function so that it can insert it in the template. The `"core-recon/recon-col"` message will need adapting accordingly, see https://openrefine.org/docs/technical-reference/translating-ui#localization-with-a-parameterized-value